q2_sample_classifier/tests/test_classifier.py

```diff
@@ -1460,6 +1460,20 @@ def test_roc_macro_average(self):
              0.7936508, 0.90476191, 1.]))
         self.assertAlmostEqual(roc_auc['macro'], 0.49930228548098726)
 
+    # these that the ROC nuts + bolts work if predictions does not have all
```

Comment:
:confused: 

```suggestion
    # Proves that the ROC nuts + bolts work if predictions does not have all
```

Reply:
LOL does not inspire confidence that everything else works does it?

I think I accidentally deleted a word at the beginning — should have been "test test that..." but your suggestion WFM, thanks!